client/src/pages/MainTable.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from "react";
+
+const MainTable = () => {
+  console.log("Page Uploaded");
+  const [clientData, SetClientData] = useState (null);
+  const [error, setError] = useState(null);
+  const [selectedClientId, setSelectedClientId] = useState (null);
+
+  useEffect(() => {
+    fetch("/api/clients")
+      .then((response) => {
+        if (!response.ok) {
+          throw Error("We could not fetch the data for that resource");
+          }
+          return response.json();
+        })
+      .then((data) => {
+        console.log(data,"After");
+        SetClientData(data);
+        console.log(clientData, "Data from useState");
+        setError(null);
+      })
+      .catch((err) => {
+              console.log("error message: ", err.message);
+              setError(err.message);
+      });
+  }, []);
+
+  console.log(clientData, "outside of fetch");
+
+if (clientData === null){
+  return <h1>Loading...</h1>;
+}
+  return (
+    <div className="container">
+      <table className="table table-sm table-striped table-hover">
+        {/* <caption>List of users</caption> */}
```

Comment:
Let's keep the `caption` so the table has a label. We can hide the caption visually by giving it a classname of `visually-hidden` from bootstrap.

The table contains our clients, so instead of `List of users` we could use `Clients` for our label.

Reply:
Already updated to Clients and visually hidden.